quarkus/service/src/test/java/org/apache/polaris/service/quarkus/it/QuarkusRestCatalogIntegrationTest.java

```diff
@@ -19,7 +19,21 @@
 package org.apache.polaris.service.quarkus.it;
 
 import io.quarkus.test.junit.QuarkusTest;
+import io.quarkus.test.junit.QuarkusTestProfile;
+import io.quarkus.test.junit.TestProfile;
+import java.util.Map;
 import org.apache.polaris.service.it.test.PolarisRestCatalogIntegrationTest;
 
 @QuarkusTest
-public class QuarkusRestCatalogIntegrationTest extends PolarisRestCatalogIntegrationTest {}
+@TestProfile(QuarkusRestCatalogIntegrationTest.Profile.class)
+public class QuarkusRestCatalogIntegrationTest extends PolarisRestCatalogIntegrationTest {
```

Comment:
Sorry, I'm a bit confused wrt #778 ... Was it not moved to `QuarkusRestCatalogIT`?

Reply:
Tests were duplicated. The only one that was removed from `test` is `QuarkusSparkIT`.

Reply:
but why duplicate them? why not just rely on the `intTest` suite?

Reply:
Let's keep. I suppose it makes sense to `test` with in-memory store and `intTest` with EclipseLink + H2